.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you use Gitflow for your project because now you are merging the **dev** branch to your **main/master** branch which you were not supposed to do in this project. You adopt this approach when you make a Group Capstone Project. But now you have a Solo Capstone Project in this project you only make one branch from development branch and after approval you merge that feature branch into dev branch :blush: